runtime/near-vm-runner/Cargo.toml

```diff
@@ -18,12 +18,16 @@ serde = { version = "1", features = ["derive"] }
 wasmer-runtime = { version = "0.18.0", features = ["default-backend-singlepass"], default-features = false, package = "wasmer-runtime-near", optional = true }
 wasmer-runtime-core = { version = "0.18.2", package = "wasmer-runtime-core-near", optional = true}
 wasmparser = "0.51"
-wasmer = { version = "1.0.2", optional = true }
-wasmer-types = { version = "1.0.2", optional = true }
-wasmer-compiler-singlepass = { version = "1.0.2", optional = true, default-features = false, features = ["std", "enable-serde"] } # disable `rayon` feature.
-wasmer-compiler-cranelift = { version = "1.0.2", optional = true }
-wasmer-engine-native = { version = "1.0.2", optional = true }
-wasmer-vm = "1.0.2"
+# wasmer = { version = "2.0.0", optional = true }
+wasmer = { git = "https://github.com/olonho/wasmer", branch = "signalless", optional = true  }
+# wasmer-types = { version = "2.0.0", optional = true }
+wasmer-types = { git = "https://github.com/olonho/wasmer", branch = "signalless", optional = true  }
+# wasmer-compiler-singlepass = { version = "2.0.0", optional = true, default-features = false, features = ["std"] } # disable `rayon` feature.
+wasmer-compiler-singlepass = { git = "https://github.com/olonho/wasmer", branch = "signalless",  optional = true }
+# wasmer-engine-universal = { version = "2.0.0", optional = true }
+wasmer-engine-universal = { git = "https://github.com/olonho/wasmer", branch = "signalless", optional = true }
+# wasmer-vm = "2.0.0"
+wasmer-vm = { git = "https://github.com/olonho/wasmer", branch = "signalless" }
```

Comment:
Nice! so the magic is there?

Reply:
Pretty much, implemented in https://github.com/wasmerio/wasmer/pull/2517/